src/engine/Filter.h

```diff
@@ -31,6 +31,10 @@ class Filter : public Operation {
   virtual void setTextLimit(size_t limit) { _subtree->setTextLimit(limit); }
 
   virtual size_t getSizeEstimate() {
+    if (_type == SparqlFilter::FilterType::REGEX) {
+      // TODO: return a better estimate
+      return std::numeric_limits<Id>::max();
+    }
     // TODO: return a better estimate
```

Comment:
While this isn't your code can you change this `TODO` as well. You can just assign it to me `TODO(schnelle)`